src/rascal/math/utils.hh

```diff
@@ -29,7 +29,7 @@
 #ifndef SRC_RASCAL_MATH_UTILS_HH_
 #define SRC_RASCAL_MATH_UTILS_HH_
 
-#include <Eigen/Dense>
+#include <Eigen/Core>
```

Comment:
Why? We never use anything else than `Dense`, do we?

Reply:
Ah, I thought it was the other way around. So then the only file that really needs `Dense` is `calculator_spherical_expansion.hh` (since it uses an eigenvalue decomposition).

Reply:
Might be, yes. I'd rather not change this in this PR and keep it for later

Reply:
Works for me.